docs/dev/plugins/cloudFoundry/CloudFoundry_Planning.md

```diff
@@ -0,0 +1,37 @@
+# Cloud Foundry(CF) - Planning
+This document show the planning and the next steps for the CF-plugin.
+
+## Playground
+To test some CF specific functionalities and to determine how the different node-types could be implemented it is useful to have a CF environment.
+Therefore we have an "test-account" on pivotal.
+For automatical testing it is probably useful to checkout how to implement CF on vSphere.
```

Comment:
Consider Running PCF Dev in a virtual machine. 
Should be possible after assigning enough ram (16 GB) and enabling the virtualisation of Hardwarevirtualisation technologies.

Just message me, if you need help